### ISSUE TITLE
minikube test workflow to use karmor release pkg

### ISSUE THREAD

**Issue by nyrahul** [kubearmor/kvm-service#27]
Currently, the GH workflow uses latest karmor code from `main` branch and this could be a problem since we can expect `main` to be broken at times!.

**Comment by seswarrajan** [comment]
Since we are enabling secure gRPC in kvm-service, I suggest to continue with the current implementation.
There is a change required in karmor vm command to use secure grpc. 
Once the next official release is ready, the change can be made. 

@nyrahul Please provide your suggestion

**Comment by seswarrajan** [comment]
The change is currently merged into dev branch.
